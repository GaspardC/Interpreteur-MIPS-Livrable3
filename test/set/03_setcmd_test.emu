#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------

set reg $3 783
