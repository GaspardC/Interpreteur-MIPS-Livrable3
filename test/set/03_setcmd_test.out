AA $3
BB 783
